scss/partials/_utility.scss

```diff
@@ -89,20 +89,20 @@
 /** 4 - Multi Column Blog Grid
 ------------------------------------------------------------*/
 .grid-col-content {
-
   &__two,
-  &__three {
+  &__three,
+  &__three-blog {
     padding-top: 0;
     width: 33.5rem;
     @include breakpoint(desktop) {
-      width: 100%
+      width: 100%;
     }
   }
 
   &__two {
     &--homepage {
       @extend .grid-col-content__two;
-      column-gap: 16.0rem !important;
+      column-gap: 16rem !important;
```

Comment:
I'm curious as to why we need `!important`.

Reply:
...hmm it was there when I found it!?

Reply:
I'm the one at fault here. Mostly due to the grid mixin.